test/make.sh

```diff
@@ -49,8 +49,20 @@ function check_terraform() {
   echo "Running terraform validate"
   #shellcheck disable=SC2156
   find . -name "*.tf" -exec bash -c 'terraform validate --check-variables=false $(dirname "{}")' \;
-  echo "Running terraform fmt"
-  terraform fmt -check=true -write=false
+}
+
+# This function lints terraform files (ending with *.tf) with hclfmt
+function lint_terraform() {
+  echo "Checking and fixing terraform formatting"
+  #shellcheck disable=SC2046
+  diff -B <(cat $(find . -name "*.tf")) <(hclfmt $(find . -name "*.tf"))
```

Comment:
I'm not familiar with `hclfmt` but I confirmed it's no longer maintained.
https://github.com/fatih/hclfmt

If we would like to adopt `hclfmt` into this repository, we need to mention it in [our README](https://github.com/GoogleCloudPlatform/terraform-google-sql-db/blob/master/README.md).

Reply:
Specifically, what's the reasoning for switching to `hclfmt` over `terraform fmt`?